ext/spl/tests/bug69181.phpt

```diff
@@ -0,0 +1,65 @@
+--TEST--
+Bug #69181 (READ_CSV|DROP_NEW_LINE drops newlines within fields)
+--FILE--
+<?php
+$filename = __DIR__ . "/bug69181.csv";
+$csv = <<<CSV
+"foo\n\nbar\nbaz",qux
+
+"foo\nbar\nbaz",qux
+CSV;
+
+file_put_contents($filename, $csv);
+
+$file = new SplFileObject($filename);
+$file->setFlags(SplFileObject::SKIP_EMPTY | SplFileObject::DROP_NEW_LINE | SplFileObject::READ_CSV);
+var_dump(iterator_to_array($file));
+
```

Comment:
Add an ``echo '===='`` or something more descriptive to make the seperation between the two dumping easier to see. Took me a good 30s to be sure the output is correct.

Reply:
Done.